### DISCUSSION TITLE
macOS Monterey support

### DISCUSSION THREAD

**Discussion by devlinford** [ohmyzsh/ohmyzsh#10495] [Help] [answered]
Hi there!

Just curious if this tool supports macOS Monterey?  I cannot see anyone with OS specific issues, but I also cannot find any minimum requirements.

Thanks in advance!

Dev

**Comment by mcornella** [comment, marked as answer]
Yes, there haven't been any reports of incompatibility.

**Reply by rvgeerligs** [reply]
I ran into this on Monterey, cannot find why:
Cloning Oh My Zsh...
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
Error: git clone of oh-my-zsh repo failed

**Reply by mcornella** [reply]
https://stackoverflow.com/a/52522566/5798232

**Reply by rvgeerligs** [reply]
installed commandline tools

**Reply by rvgeerligs** [reply]
ohmyzsh works

**Comment by devlinford** [comment]
Great!

Thank you!

**Comment by rvgeerligs** [comment]
I ran into this:
Cloning Oh My Zsh...
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
Error: git clone of oh-my-zsh repo failed
